files/zuul-tests.yaml

```diff
@@ -12,6 +12,11 @@
       state: present
     become: true
 
+  - name: "set up git: email"
```

Comment:
Is it needed to duplicate it in `files/install-requirements.yaml` and in `files/zuul-tests.yaml`?
What about to have it in one file?

Reply:
Yeah, I tried to tell Honza exactly that - would be nice to merge those two playbooks - I am not doing that in this PR.